README.md

```diff
@@ -1,12 +1,11 @@
 # Kit's Architecture 
 ## Introduction: 
   
-   When migrating to a new package dependency manager in Xcode, there are several challenges that developers may face. This document outlines some of the potential issues and the struggles we faced during the development of migrating them.
+   When migrating to a swift package manager in Xcode, there are several challenges that developers may face. This document outlines some of the potential issues and the struggles we faced during the development of migrating them.
```

Comment:
1. Swift Package Manager is not the correct term. Swift Packages is correct term
2. Development and migration both are different tracks. We faced during the migration is the correct term

Reply:
done